handler/cookie.go

```diff
@@ -0,0 +1,46 @@
+package handler
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/kodekulture/wordle-server/internal/config"
+	"github.com/lordvidex/x/auth"
+)
+
+func isProd() bool {
```

Comment:
can we make this constant? i.e. `isSecured := config.Get("ENV") == "prod"`, `env` won't change during runtime